Lib/fontParts/base/bPoint.py

```diff
@@ -21,16 +21,16 @@ def _setPoint(self, point):
 
     def __eq__(self, other):
         if hasattr(other, "_point"):
-            return self._point == other._point
-        return NotImplemented
+            return id(self._point) == id(other._point)
```

Comment:
In that case, looks like "return self._point is other._point" is what you want. You want to return True only if the two _point members point to the same object. Right?

Reply:
Yup, that's the idea, but it's not quite working yet. I'll keep hammering it...